api/src/App/Routes.php

```diff
@@ -93,6 +93,38 @@ function () use ($app, $authMiddleware) {
     }
 )->add(new App\Middleware\CiabMiddleware($app))->add($authMiddleware);
 
+$app->group(
+    '/payment',
+    function () use ($app, $authMiddleware) {
+        $app->add(function ($request, $response, $next) {
+            \Stripe\Stripe::setApiKey($_ENV['STRIPE_PRIVATE_KEY']);
+            return $next($request, $response);
+        });
+        $app->post('/checkout', 'App\Controller\Payment\PostCheckout');
+    }
+)->add(new App\Middleware\CiabMiddleware($app))->add($authMiddleware);
+
+$app->add(function ($request, $response, $next) {
+    \Stripe\Stripe::setApiKey($_ENV['STRIPE_PRIVATE_KEY']);
+    return $next($request, $response);
+})->post('/vendor/stripe/webhook', 'App\Controller\Vendor\Stripe\PostWebhook');
+
+$app->group(
+    '/stores',
+    function () use ($app, $authMiddleware) {
+        $app->get('[/]', 'App\Controller\Stores\ListStores');
+        $app->get('/{id}', 'App\Controller\Stores\GetStore');
+        $app->post('[/]', 'App\Controller\Stores\PostStore');
+        $app->put('/{id}', 'App\Controller\Stores\PutStore');
+        $app->delete('/{id}', 'App\Controller\Stores\DeleteStore');
+        $app->get('/{store_id}/products', 'App\Controller\Stores\ListProducts');
+        $app->post('/{store_id}/products', 'App\Controller\Stores\PostProduct');
+        $app->get('/{store_id}/products/{id}', 'App\Controller\Stores\GetProduct');
+        $app->put('/{store_id}/products/{id}', 'App\Controller\Stores\PutProduct');
+        $app->delete('/{store_id}/products/{id}', 'App\Controller\Stores\DeleteProduct');
```

Comment:
hummmmmm. I can go either way here... But me current standard is to not use the plural of the resource. 
member, department, etc... so product**s**  and store**s** is a bit of a break from that.

Reply:
The Internet-wide standard, however, is plural.

Reply:
really, How did i miss that in my restful api research.  Again my restful api research was mostly about 3-4 years ago. Is there a standards docuemt somewhere I should read? Well sounds like we need to update all the rest of the API.

Reply:
I'm not sure I've ever seen it written down, and it's always possible it's more a Rails-specific interpretation of RESTFUL conventions than any formal standard, but in 10 years doing this, I've always seen these as plural.

Reply:
ok we can fight that fight however. But I do crave consistency. The open source developer in my says "Always conform to the standards of the existing product" But I will cut you some slack here.